source/win-quicklook/QuicklookEditor.vue

```diff
@@ -182,7 +182,8 @@ export default defineComponent({
             math: (global as any).config.get('display.renderMath'),
             tasks: (global as any).config.get('display.renderTasks'),
             headingTags: (global as any).config.get('display.renderHTags'),
-            tables: (global as any).config.get('editor.enableTableHelper')
+            tables: (global as any).config.get('editor.enableTableHelper'),
+            codeBlockButton: (global as any).config.get('display.renderCodeBlockButton')
```

Comment:
The quicklook editor has been nuked in the meantime, so you can remove this change.